scripts/openyurt-deployer/configs/kubeTemplate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+clusters:
+- cluster:
+    server: http://127.0.0.1:10261
```

Comment:
what is this address:port?

Reply:
i have double check this. this is the default template in openyurt to reconfigure the kubelet